File: Services/Dto/CustomerDto.cs
using System;
using System.Collections.Generic;
using Services.Dto.CollectsDelivers;
using Services.Dto.Outsourced;
using Services.Dto.ServiceBudgetBench;
using Services.Dto.Shared;

namespace Services.Dto
{
    public class CustomerDto
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string CNPJ { get; set; }
        public string Responsible { get; set; }
        public string Comments { get; set; }
        public bool Assured { get; set; }
        public bool CustomerType { get; set; }
        public decimal Payment { get; set; }
        public DateTime Registered { get; set; }
        public int Expiration { get; set; }
        public bool Disabled { get; set; }
        public bool ToBusinessBox { get; set; }
        public decimal Discount { get; set; }
        public AddressDto Address { get; set; }
        public ContactDto Contact { get; set; }
        public virtual List<EletronicRepairDto> EletronicsRepairs { get; set; }
        public virtual List<ServiceBudgetDto> ServicesBudgets { get; set; }
        public virtual List<ServiceBenchDto> ServicesBenchs { get; set; }
        public virtual List<CollectDeliverDto> CollectsDelivers { get; set; }

    }
}

File: Views/src/components/customer/dto/customer-dto.ts
import { ContactDto } from 'src/shared/dtos/contact-dto';
import { AddressDto } from 'src/shared/dtos/address-dto';
import { NetworkDevicesDto } from '../technician/infra/dto/network-devices-dto';
import { ServiceBudgetDto } from 'src/components/services-provision/budget/dto/service-budget-dto';
import { CollectDeliverDto } from 'src/components/out-sourced/collect-deliver-list-table-all/dto/collect-deliver-dto';
export class CustomerDto {
  id: number;
  name: string;
  cnpj: string;
  responsible: string;
  assured: boolean;
  customerType: boolean;
  payment: number;
  registered: Date;
  expiration: number;
  disabled: boolean;
  toBusinessBox: boolean;
  discount: number;
  addressid: number;
  address: AddressDto;
  contactid: number;
  contact: ContactDto;
  comments: string;
  netWorkDevices: NetworkDevicesDto[];
  servicesBudgets: ServiceBudgetDto[];
  sourceCollectDelivers: CollectDeliverDto[];
  destinyCollectDelivers: CollectDeliverDto[];

}
